Background/scripts/control_regions.py

```diff
@@ -106,7 +109,9 @@ def main(f_in, cat, f_out):
     #dy_norm = ROOT.RooRealVar("bkg"+suffix+"_norm","bkg"+suffix+"_norm",1-bkg_frac.getVal(),0,n_events)
 
     c = ROOT.TCanvas("canvas", "canvas", 600, 600)
-    frame.SetAxisRange(65.0,120)
+
+    if (my>=80) and (my<=100):
+        frame.SetAxisRange(65.0,120)
```

Comment:
Do we want this to be controlled by `low_bound` and `high_bound` or not?

Reply:
I'd like to keep them distinct since in principal they control two different things, fitting range and plotting range.